### ISSUE TITLE
Errors skip in this library!

### ISSUE THREAD

**Issue by monoflash** [360EntSecGroup-Skylar/excelize#539]
The library is written with a error skip at all.
Because of this, excel files do not open without error messages.

I was faced with the fact that the xml crashes with an error when the openned excel file in an encoding different from UTF-8, but the library ignores this error not reporting the error in any way.

After looking at the code, I found error skipping almost everywhere, and as a result, at the first not significant error everything stops working.

To my regret, I have to abandon the use of this library, but I really hope that all the error skips will be processed and the library can be usable again.

**Comment by xuri** [comment]
Hi @monoflash, thanks for your issue. Please provide .xlsx file attachment that created in encoding different from UTF-8 if you can.

**Comment by monoflash** [comment]
This is a financial report provided by our contractors. When I delete all confidential information from it and save it, it is transcoded to UTF-8. And along with the financial information, I can’t provide the file.
I will try later to prepare an example file that can be provided to you.

**Comment by monoflash** [comment]
I manually edited the XML file contained in xlsx.
file: xl/worksheets/sheet1.xml

If you look at it, then its code page is not UTF-8, for such XML files, the XML library requires a transcoder.

The fact that XML file is not encoded in UTF-8 is normal, Microsoft Excel accepts this well.

The only thing I did was manually delete the data from XML, so when open this xlsx, there will be a message about data corruption.
I think this is enough for an example.

[m2.xlsx](https://github.com/360EntSecGroup-Skylar/excelize/files/3980870/m2.xlsx)
